lib/ae_mdw_web/controllers/aex9_controller.ex

```diff
@@ -310,7 +311,13 @@ defmodule AeMdwWeb.Aex9Controller do
   defp balance_reply(%Conn{assigns: %{state: state, opts: opts}} = conn, contract_pk, account_pk) do
     {amount, {type, height, hash}} =
       if top?(opts) do
-        DBN.aex9_balance(contract_pk, account_pk, top?(opts))
+        case DBN.aex9_balance(contract_pk, account_pk, top?(opts)) do
+          {:ok, account_balance} ->
+            account_balance
+
+          {:error, unavailable_error} ->
```

Comment:
Can we handle all errors in a friendly manner to users? Following the `Runner.call_error()` type we could do something like
```elixir
{:error, :contract_does_not_exist} -> {:error, ErrInput.Aex9BalanceNotAvailable.exception(value: contract_pk)}
{:error, :dry_run_error} -> {:error, ErrInput.Aex9BalanceNotAvailable.exception(value: "dry run error")}
{:error, reason} -> {:error, ErrInput.Aex9BalanceNotAvailable.exception(value: reason)}
```

Reply:
I think exceptions shouldn't be used without raising them but this is sth that needs to be chored throughout the code. Added for a prefixed user message.